src/Cms/Blueprint.php

```diff
@@ -367,10 +367,14 @@
 				];
 			}
 
-			$columns[$columnKey] = array_merge($columnProps, [
+			$columns[$columnKey] = [
+				...$columnProps,
 				'width'    => $columnProps['width'] ?? '1/1',
-				'sections' => $this->normalizeSections($tabName, $columnProps['sections'] ?? [])
-			]);
+				'sections' => $this->normalizeSections(
+					$tabName,
+					$columnProps['sections'] ?? []
+				)
```

Comment:
@lukasbestle seems like Psalm doesn't like array unwrapping that overwrites keys :/

Reply:
There's an issue for this here and it was closed as intentional: https://github.com/vimeo/psalm/issues/8616
They do have a point that the unpacking syntax could lead to cases where keys are overwritten accidentally, but it's tedious to suppress these warnings all the time.

TBH I'm not sure if the unpacking syntax is a cure-all solution and good replacement for `array_merge()` anyway. The three dots are very easy to miss compared to the explicit `array_merge()`. I would keep most of the `array_merge()` uses.

Reply:
Hmm I disagree, I think it would make our code styles between PHP and JavaScript align much more and code easier to read. The three dots merging way has proven itself as very helpful in JavaScript, so why wouldn't that work as well for PHP?

And I would suggest to add the error to the global Psalm ignore list. As overwriting keys really is a use case here, not an issue.